wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -216,6 +217,8 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
             dashPayProfileDao.loadAll().forEach {
                 platform.stateRepository.addValidIdentity(it.userIdentifier)
             }
+
+            platform.stateRepository.storeIdentity(blockchainIdentity.identity!!)
```

Comment:
This is where we could add other documents that we created, but at this point it doesn't seem necessary to modify DB tables to save those documents.